README.md

```diff
@@ -22,11 +22,11 @@ hugo server
 
 Feel free to submit pull requests, make issues, and edit anything and everything.
 
-Anyone can add a post to the blog. We'll figure out some way of organising that as it goes. Just copy the format of an existing post and submit a pull request.
-
-Anybody can add a post to the blog. Just create a new file in `content/blog/`,
-probably prefix it with a year if its a recurring event or something. You can
-just copy the definition at the top from another file.
+Anyone can add a post to the blog. We'll figure out some way of organising that
+as it goes. Just copy the format of an existing post and submit a pull
```

Comment:
When I meant they should be merged, I didn't just mean stuck together 😛 The two paragraphs are near duplicates ("Anyone can add a post to the blog", then later, "Anybody can add a post to the blog").

Just de-dupe, and go with something like:

"Anyone can add a post to the blog. Just create a new file in `content/blog`, probably prefix it with a year if it's a recurring event or something. You can just copy the definition at the top from another file. Then submit a pull request."